tests/test_open.py

```diff
@@ -317,8 +282,7 @@ def test_filename_matches_meta(tmp_path, model):
     # Show datamodels.open will update the filename in memory
     with (
         pytest.warns(
-            match="meta.filename: test_filename.asdf does not match filename: "
-            "test_filename_read.asdf, updating the filename in memory!"
+            match="meta.filename: \\? does not match filename: test_filename_read.asdf, updating the filename in memory!"
```

Comment:
What does "\\\\?" indicate?

Reply:
@mairanteodoro is spot on. For some context here, the fake string value is "?". In this test the raw node is written to disk (the node doesn't turn into a datamodel and isn't subject to the usual `meta.filename` overwriting). This is used to produce a file where the `meta.filename` (for the updated test "?") doesn't match the actual filename ("test_filename_read.asdf").